ext/stringio/stringio.c

```diff
@@ -1160,7 +1161,9 @@ prepare_getline_args(struct getline_arg *arg, int argc, VALUE *argv)
 	    keywords[0] = rb_intern_const("chomp");
 	}
 	rb_get_kwargs(opts, keywords, 0, 1, &vchomp);
-	arg->chomp = (vchomp != Qundef) && RTEST(vchomp);
+        if (respect_chomp) {
+	    arg->chomp = (vchomp != Qundef) && RTEST(vchomp);
```

Comment:
Q: is GitHub rendering this indent wrong for me or is it not indented?

Reply:
I think it's indented correctly.  The render issue is caused by the tab vs. space issue (spaces for new code, tabs for old code).

Reply:
Thanks, today I learned!